tensorflow/examples/learn/iris_custom_decay_dnn.py

```diff
@@ -11,6 +11,8 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+"""Example of DNNClassifier for Iris plant dataset,with exponential decay"""
```

Comment:
nit: Per Google Python Style (https://google.github.io/styleguide/pyguide.html), there needs to be
1) A space after the comma,https://google.github.io/styleguide/pyguide.html
2) A period after the last word.

Also make sure that the line is <=80 chars wide as you make further changes.